templates/project/rector.php.twig

```diff
@@ -20,7 +20,7 @@ declare(strict_types=1);
 use Rector\Config\RectorConfig;
 use Rector\Php70\Rector\FunctionLike\ExceptionHandlerTypehintRector;
 use Rector\Php71\Rector\FuncCall\CountOnNullRector;
-{% if (project.name == 'block-bundle' and branch.name == '4.x') or (project.name == 'entity-audit-bundle' and branch.name == '1.x') %}
+{% if (project.name == 'block-bundle' and branch.name == '4.x') or (project.name == 'entity-audit-bundle' and branch.name == '1.x') or (project.name == 'admin-bundle' and branch.name == '4.x') %}
```

Comment:
The more we add checks, and the more I wonder it wouldn't be simpler to ignore this rule in a general way.

Reply:
I dont think there will be more cases.